gateway/engine/core/src/main/java/io/apiman/gateway/engine/impl/DefaultPolicyErrorWriter.java

```diff
@@ -15,6 +15,7 @@
  */
 package io.apiman.gateway.engine.impl;
 
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Hmm, we should create an IntelliJ and Eclipse standard profile to prevent this import thrashing (just reordering of imports by IDE, no actual real changes). Note to self!